dev/CommonStyles/TestUI/MenuFlyoutPage.xaml.cs

```diff
@@ -12,15 +13,33 @@ namespace MUXControlsTestApp
     [TopLevelTestPage(Name = "MenuFlyout", Icon = "MenuFlyout.png")]
     public sealed partial class MenuFlyoutPage : TestPage
     {
+        private MenuFlyout sharedFlyout;
         public MenuFlyoutPage()
         {
             this.InitializeComponent();
+
+            sharedFlyout = (MenuFlyout)Resources["SampleContextMenu"];
         }
 
         private void TestMenuFlyoutItemClick(object sender, object e)
         {
             TestMenuFlyoutItemHeightTextBlock.Text = $"{TestMenuFlyoutItem.ActualHeight}";
             TestMenuFlyoutItemWidthTextBlock.Text = $"{TestMenuFlyoutItem.ActualWidth}";
         }
+
+        private void Grid_ContextRequested(UIElement sender, ContextRequestedEventArgs args)
+        {
+            Point point;
+            var requestedElement = sender as FrameworkElement;
+
+            if (args.TryGetPosition(requestedElement, out point))
```

Comment:
nit: You could also use the following and remove line 32:

```suggestion
            if (args.TryGetPosition(requestedElement, out Point point))
```